tests/unit/test_job_submitter.py

```diff
@@ -19,162 +23,258 @@
 
 @pytest.mark.parametrize("maskfile", [True, False])
 def test_job_creation_and_localrun(tmp_path, maskfile):
-    args, setup = _create_setup(tmp_path, mask_file=maskfile)
-    setup.update(run_local=True)
-    job_submit(**setup)
-    _test_output(args)
+    """ Tests that the jobs are created and can be run locally 
+    from mask-string and mask-file. """
+    setup = InputParameters(working_directory=tmp_path, run_local=True)
+    setup.create_mask(as_file=maskfile)
+    job_submit(**asdict(setup))
+    _test_output(setup)
+
+
+def test_output_directory(tmp_path):
+    """ Tests that the output is copied to the output destination. 
+    As a by product it also tests that the jobs are created and can be run locally. """
+    setup = InputParameters(
+        working_directory=tmp_path, 
+        run_local=True,
+        output_destination=tmp_path / "my_new_output" / "long_path",
+    )
+    setup.create_mask()
+    job_submit(**asdict(setup))
+    _test_output(setup)
+
+
+def test_wrong_uri(tmp_path):
+    """ Tests that wrong URI's are identified. """
+    setup = InputParameters(
+        working_directory=tmp_path, 
+        run_local=True,
+        output_destination="root:/eosuser.cern.ch/eos/my_new_output",
+    )
+    setup.create_mask()
+    with pytest.raises(ValueError) as e:
+        job_submit(**asdict(setup))
+    assert "EOS-URI" in str(e)

```

Comment:
This tests for a wrong `EOS` uri, maybe we can rename it. How about `test_raises_on_wrong_eos_uri`?